spec/transaction/paths.yaml

```diff
@@ -789,6 +942,249 @@ paths:
+                    status: completed
+                    number_parts: 1
+                    created: "2019-08-24T14:15:22Z"
+                    completed: "2019-08-24T14:15:22Z"
+                    file:checksum: 12200ADEC47F803A8CF1055ED36750B3BA573C79A3AF7DA6D6F5A2AED03EA16AF3BC
+                  - upload_id: KrFTuglD.N8ireqry_w3.oQqNwrYI7SfSXpVRiusKah0YigDnuM06hfJNIUZg4R_No0MMW9FLU2UG5anTW0boTUYVxKfBZWCFXqnQTpjnQEo1K7la39MYpjSTvIbZgnG
+                    status: aborted
+                    number_parts: 1
+                    created: "2019-08-24T14:15:22Z"
+                    aborted: "2019-08-24T14:15:22Z"
+                    file:checksum: 12200ADEC47F803A8CF1055ED36750B3BA573C79A3AF7DA6D6F5A2AED03EA16AF3BC
+                links:
+                  - rel: next
+                    href: https://data.geo.admin.ch/api/stac/v0.9/collections/ch.swisstopo.pixelkarte-farbe-pk50.noscale/items/smr200-200-4-2019/assets/smr50-263-2016-2056-kgrs-2.5.tiff/uploads?cursor=0d34
+        "400":
+          $ref: "../components/responses.yaml#/components/responses/BadRequest"
+        "404":
+          $ref: "../components/responses.yaml#/components/responses/NotFound"
+        "5XX":
+          $ref: "../components/responses.yaml#/components/responses/ServerError"
+    post:
+      tags:
+        - Collection Asset Upload Management
+      summary: Create a new collection asset multipart upload
+      description: |
+        Create collection asset multipart upload.
+
+        A file part must be at least 5 MB except for the last one and at most 5 GB, otherwise the complete operation will fail.
+
+        *Note: in order to provide integrity check during the upload, the base64-encoded 128-bit MD5 digest of each part must be
+        computed and passed in the create endpoint. Then this digest must also be passed as `Content-MD5` header during the upload.*
+      operationId: createCollectionAssetUpload
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "./components/schemas.yaml#/components/schemas/assetUploadCreate"
+      responses:
+        201:
+          description: Created collection asset multipart upload
+          headers:
+            Location:
+              description: A link to the collection asset multipart upload object
+              schema:
+                type: string
+                format: url
+          content:
+            application/json:
+              schema:
+                $ref: "./components/schemas.yaml#/components/schemas/assetUploadCreate"
+        "400":
+          $ref: "../components/responses.yaml#/components/responses/BadRequest"
+        "404":
+          $ref: "../components/responses.yaml#/components/responses/NotFound"
+        "409":
+          description: Another upload is already in progress, you need first to complete or abort it before creating a new upload.
```

Comment:
Shouldn't this be "you first need to complete" instead of "you need first to complete"?

Reply:
Fixed here, thanks. Would you mind correcting also on line 660 where this is probably copied from?